### ISSUE TITLE
CHANGELOG out of date

### ISSUE THREAD

**Issue by NiklasBr** [magento/magento2#28021]
### Summary (*)

[CHANGELOG](https://github.com/magento/magento2/blob/2.4-develop/CHANGELOG.md) was updated last time on 8 Oct 2019 for version 2.3.3, since then version 2.3.4 has been released on 28 Jan 2020 and just now version 2.3.5.

Neither version has been documented in the document.

### Examples (*)

See [CHANGELOG](https://github.com/magento/magento2/blob/2.4-develop/CHANGELOG.md)

### Proposed solution

Please do :)

**Comment by NiklasBr** [comment]
@magento Yes, I confirm

**Comment by hostep** [comment]
You can view it over here: https://github.com/magento/magento2/blob/2.3.5/CHANGELOG.md

I currently have no opinion if the changelog should be updated as well on the `2.4-develop` branch.

**Comment by NiklasBr** [comment]
Still relevant

**Comment by sdzhepa** [comment]
Hello @NiklasBr and @hostep 

Thank you for your collaboration and contribution!

At some point, Magento stoped the update changelog in '2.4-develop' for 2.3.x.
It was done intentionally because branches have a different commit history.

But actual changelog can be found in branch/tag per specific version as it was mentioned https://github.com/magento/magento2/issues/28021#issuecomment-621129512

So, this issue can be closed.

Additionally, thank you to raise this issue! I noticed that `2.4-develop` was not updated with 2.4.1 and 2.4.2 data. I have already created an internal ticket to fix it and it should be done soon
